libhwmon/CMakeLists.txt

```diff
@@ -6,10 +6,14 @@ set(SOURCES
 )
 
 add_library(hwmon ${SOURCES})
-target_link_libraries(hwmon devcore)
+if (UNIX)
+	target_link_libraries(hwmon devcore dl)
```

Comment:
I suggest using CMake's list APPEND before target_link_libraries.

Reply:
@reagentoo Sorry, not really CMake user, would this new CR proposal works? At least it compiled to me